src/components/Oauth/WorkspaceEntry/WorkspaceOauthFlow.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * OAuth flow for any providers that do not require the consumer to enter a subdomain first.
+ */
+
+import { useCallback, useState } from 'react';
+
+import { useApiKey } from '../../../context/ApiKeyContextProvider';
+import { useProject } from '../../../context/ProjectContextProvider';
+import { capitalize } from '../../../utils';
+import { fetchOAuthCallbackURL } from '../fetchOAuthCallbackURL';
+import OAuthPopup from '../OAuthPopup';
+
+import { WorkspaceEntry } from './WorkspaceEntry';
+
+interface NoSubdomainOauthFlowProps {
```

Comment:
This should be `WorkspaceOauthFlowProps`

Reply:
fixed in later pr.